Resources/translations/fr/systeminfo.fr.xlf

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <body>
+      <trans-unit id="34cebbd8a2d90255fd1317160dedb480" resname="system.information">
+        <source>system.information</source>
+        <target>Information système</target>
+      </trans-unit>
+      <trans-unit id="cf1e8c14e54505f60aa10ceb8d5d8ab3" resname="server">
+        <source>server</source>
+        <target>Serveur</target>
+      </trans-unit>
+      <trans-unit id="27fe419d5eb953490a9fd3132c79aa2e" resname="ez.platform.version">
+        <source>ez.platform.version</source>
+        <target>Vous utilisez la version Easy Platform</target>
```

Comment:
*Vous utilisez eZ Platform version* (there's something after this sentence, most likely the translation should not be organized this way)